blaze/tests/test_interactive.py

```diff
@@ -201,6 +201,19 @@ def test_table_resource():
         assert isinstance(t.data, CSV)
         assert into(list, compute(t)) == into(list, csv)
 
+def test_explicit_dshape():
+    ds = dshape("var * {a: ?float64, b: ?string, c: ?float32}")
+    s = """a,b,c
```

Comment:
I never knew there was a `textwrap` module -- that's awesome, thanks for the pointer.